python/src/main/processing/filter_dataset.py

```diff
@@ -0,0 +1,54 @@
+"""
+This script allows the user to filter the dataset.
+Currently is supported only filter by allowed extensions list.
+
+It accepts
+    * path to a folder with input -- folder with projects
+    * path to a folder for output
+    * optional argument --extension-list with a list allowed file extensions
+    (without this argument all files will be saved).
```

Comment:
Let's mention that filtered out files will be deleted completely when input==output.